hackathon/urls.py

```diff
@@ -7,18 +7,22 @@
     HackathonDetailView,
     enroll_toggle,
     judging,
-    check_projects_scores
+    check_projects_scores,
+    view_hackathon,
 )
 
 urlpatterns = [
     path('', HackathonListView.as_view(), name="hackathon-list"),
     path("<int:hack_id>/team/<int:team_id>/judging/", judging, name="judging"),
     path("<int:hack_id>/final_score", check_projects_scores, name="final_score"),
     path("create_hackathon", create_hackathon, name='create_hackathon'),
-    path("<int:hackathon_id>/update_hackathon", update_hackathon, name="update_hackathon"),
-    path("<int:hackathon_id>/delete_hackathon", delete_hackathon, name="delete_hackathon"),
-    path('<int:pk>/',
-         HackathonDetailView.as_view(), name='hackathon_detail'),
-    path('enroll', enroll_toggle, name='enroll_toggle')
+    path("<int:hackathon_id>/", view_hackathon,
+         name='view_hackathon'),
+    path("<int:hackathon_id>/update", update_hackathon,
+         name="update_hackathon"),
+    path("<int:hackathon_id>/delete", delete_hackathon,
+         name="delete_hackathon"),
+    # path('<int:pk>/',
```

Comment:
Do we still need this, if line 19 is being added now for `view_hackathon`?

Reply:
Removed this as well.